eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Rules/TestChain.hs

```diff
@@ -289,12 +321,15 @@ adaPreservationChain ::
   Property
 adaPreservationChain =
   forAllChainTrace @era longTraceLen $ \tr -> do
-    let ssts = sourceSignalTargets tr
+    let ssts :: [SourceSignalTarget (CHAIN era)]
+        -- Signal(CHAIN era) = Block (BHeader (EraCrypto era)) era
```

Comment:
dead?

Reply:
Just reay a comment, so I changed it to ook like this

```
let ssts :: [SourceSignalTarget (CHAIN era)]
        -- In this test, the STS Signal has this definition
        -- Signal(CHAIN era) = Block (BHeader (EraCrypto era)) era
        ssts = sourceSignalTargets tr
```